src/components/ProgressIndicator/index.js

```diff
@@ -82,7 +82,7 @@ const stepLabels = {
 };
 
 
-const ProgressIndicator = ({ title, submission=null, steps, submissionAllowed }) => {
+const ProgressIndicator = ({ title, submission=null, steps, submissionAllowed, completed=false }) => {
```

Comment:
PropTypes to update :slightly_smiling_face:

Reply:
this time I didn't miss it, it's on line 191 :smile: